### ISSUE TITLE
Score doesn't scale when adding / removing mods

### ISSUE THREAD

**Issue by forteus19** [MinisBett/rxhddt#3]
I found that when you add or remove mods to a replay, the score displayed on the scoreboard doesn't change to the new value.

**Comment by minisbett** [comment]
Sadly this is not possible due to technical reasons. There is no "global formula" for applying the mod multiplier. The mod multipler is being applied on the score you get per hit circle click. This means that I would need to simulate the whole replay and check what circles the user hit. But this is both hard and not 100% working because sometimes replays are not the same as you playing because replays are recorded in 60fps and if you are unlucky you hit a note between those replay frames. In this case there is no way to restore the original play performance.

TL;DR: It's not possible because score multiplier is way harder to apply than it sounds like.
